### ISSUE TITLE
_WD_CheckContext - Return values

### ISSUE THREAD

**Issue by mlipok** [Danp2/WebDriver#126]
is this following description :

```
;                  Failure - Returns one of the following values:
;                  |$_WD_STATUS_Invalid (0) - Browser context is invalid
;                  |$_WD_ERROR_Exception

```

wrong  or proper ?

$_WD_ERROR_Exception should not be returned as value but as @error

**Comment by Danp2** [comment]
Yes, I believe that to be incorrect. I should state that it returns $_WD_STATUS_Invalid and sets @error equal to $_WD_ERROR_Exception

**Comment by mlipok** [comment]
Do it please

**Comment by Danp2** [comment]
Fixed in aa173f3a51494dc77b4359b5920138fe5afc1edb
